### PR TITLE
Vet clinic DATABASE: add "Join Table" for visits 🚀 

### PR DESCRIPTION
**Pull request by evansnyamekye**
**Vet Clinic Database: Add "Join Table" for visits**

This pull request introduces a crucial enhancement to the Vet Clinic database by implementing a "join table" to handle the relationship between animals and vets during visits. The new visits table provides a structured way to track animal visits to vets, including the date of the visit. 

**Changes Made:**

- Created the visit table to manage the many-to-many relationship between animals and vets during visits 👍.
- Added columns to store animals_id, vet_id, and visit_date 💯.
- Included an auto-incremented id column as the primary key for each visit record ♾️. 

This update adds a layer of organization to the data, making it easier to manage and retrieve information about animal visits. 
Below shows screenshot of the new visits table and sample queries have been included in the pull request for verification.  
1.
j1
2.
j2
3.
j3
4.
j4
5.
j5
6.
j6
7.
j7
8.
j8
9.
j10
10
j9
11
j10

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -101,4 +101,61 @@ FROM owners
 LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.full_name
 ORDER BY animal_count DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
```
Comment (t-yanick) [inline queries.sql:105]: - [ ] Great job working on this milestone. You are almost there.
- At the moment, you are missing the query; `Who was the last animal seen by William Tatcher?` Kindly add this query, run it, and add the screenshot to the PR description body.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @evansnyamekye , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  SQL commands put in the correct files
:heavy_check_mark: Query results screenshots added to PR body
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Use of correct gitflow

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (evansnyamekye) [reply queries.sql:105]: @t-yanick `Who was the last animal seen by William Tatcher?` Added as requested 

j10

Comment (evansnyamekye) [reply queries.sql:105]: All corrections done as requested by Code Reviewer 💯. 

Thank you 😃.

File: queries.sql [hunk 2]
```diff
@@ -101,4 +101,68 @@ FROM owners
...
+SELECT animals.name AS animal_name
+FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.visit_date DESC
+LIMIT 1;
```
Comment (mukhammadrizooff) [inline queries.sql:112]: - [ ] [REQUIRED] Good job so far! I appreciate what you have done but the [previous reviewer mentioned](https://github.com/evansnyamekye/vet-clinic/pull/4#discussion_r1372656202) that the result of  `query; Who was the last animal seen by William Tatcher?` screenshot must picked to `Pull Request Description` not a put `comment` of request 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @evansnyamekye, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (evansnyamekye) [reply queries.sql:112]: All corrections done as requested by Code Reviewer 💯. 
Screen shot picked to `Pull Request Description`

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @evansnyamekye 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the requested changes from the previous reviewers. 💯 

Cheers, and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**